Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -2119,6 +2121,15 @@ - (void)_dispatchSetDeviceOrientation:(WebCore::IntDegrees)deviceOrientation
     _perProcessState.lastSentDeviceOrientation = deviceOrientation;
 }
 
+- (void)_dispatchSetOrientationForMediaCapture:(WebCore::IntDegrees)orientationForMediaCapture
+{
+    if (_perProcessState.lastSentOrientationForMediaCapture && _perProcessState.lastSentOrientationForMediaCapture.value() == orientationForMediaCapture)
+        return;
+
+    _page->setOrientationForMediaCapture(orientationForMediaCapture);
+    _perProcessState.lastSentOrientationForMediaCapture = orientationForMediaCapture;
```

Comment:
I would tend to set _perProcessState.lastSentOrientationForMediaCapture then call setOrientationForMediaCapture.

Reply:
That would differ from all of the other methods above :)